data/cube/View.js

```diff
@@ -228,11 +242,11 @@ export class View extends HoistBase {
 
     // Generate a new full data representation
     generateRows() {
-        const {query, cube} = this,
+        const {query, _aggContext} = this,
             {dimensions, includeRoot} = query,
             rootId = 'root';
 
-        const records = cube.store.records.filter(rec => query.test(rec));
+        const records = Array.from(_aggContext.filteredRecords.values());
```

Comment:
Should the AppContext return an array directly here via a getter, and move filteredRecord to _filteredRecords?